tests/forms/uploadfield/UploadFieldTest.php

```diff
@@ -167,7 +167,9 @@ public function testAllowedExtensions() {
 			'UploadFieldTest_Controller/Form/field/AllowedExtensionsField/upload',
 			array('AllowedExtensionsField' => $this->getUploadFile($invalidFile))
 		);
-		$this->assertTrue($response->isError());
+		$responseJson = Convert::json2array($response->getBody());
+		
+		$this->assertTrue($response->isError() || isset($responseJson[0]['error']));
```

Comment:
Yeah, that makes sense. Given one of the other files in the commit actually removed setting the error status code (that `$response->isError()` would pick up), in terms of the test the `isset` check is the valid one. Will fix that in another commit shortly.